dali/pipeline/operators/crop/crop.cu

```diff
@@ -23,7 +23,7 @@ __global__ void BatchedCropKernel(const int C, const int* height, const int* wid
                                   const uint8 *const *img_ptrs,
                                   const int *in_strides,
                                   DALITensorLayout layout, Out *out) {
-  const int n = blockIdx.x;
+  const int n = blockIdx.x + threadIdx.x;
```

Comment:
WIP: Need to find the way to retrieve the correct N to produce the required W and H for the crop.

At the moment, we have an illegal memory access in L53

Reply:
Input or the output?

Reply:
output in this case

Reply:
Issue was resolved a couple of lines below. Output ptr was calculated using W and H (fallout from when we only used fixed W and H). Currently the offset is calculated based on the values for widths and heights